code/game/gamemodes/game_mode.dm

```diff
@@ -346,14 +346,41 @@ var/global/list/additional_antag_types = list()
 			else if(isghost(M))
 				ghosts++
 
+		if(M.get_preference_value(/datum/client_preference/play_admin_midis) == GLOB.PREF_YES)
+			if (escaped_total < 1)
```

Comment:
these values need to be percentages of historic crew count, that number is already tracked in the code that calculates when enough people have died to trigger an auto ending

Reply:
West wanted it this way, actually.